extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/security/Form.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.vertx.http.runtime.security;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Selects {@link FormAuthenticationMechanism}.
+ * Equivalent to '@HttpAuthMechanism("form")'.
+ *
+ * @see HttpAuthMechanism for more information
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ ElementType.TYPE, ElementType.METHOD })
+public @interface Form {
```

Comment:
```suggestion
public @interface FormAuthentication {
```

This is probably where it makes real sense, because there are also JAX-RS Form parameters